docs/advanced-topics/tools-vs-workflows.rst

```diff
@@ -0,0 +1,117 @@
+Tools vs Workflows: Detailed Comparison
+=======================================
+
+.. tip::
+  Generally speaking, if you care more about the Docker image than the descriptor file, and are using CWL, you are dealing with a tool. If you care more about the descriptor file, you are dealing with a workflow.
+
+History
+-------
+When Dockstore was created, CWL was the first descriptor language we supported. CWL has a very clear distinction between a tool and a workflow. However, our definition for each does not completely align with the language's specification -- strictly speaking, a CWL CommandLineTool does not require a Docker image, but we require tools to be associated with a Docker image on Dockstore. Furthermore, while other descriptor languages do not have separate concepts for tools and workflows, we still maintain a distinction between a tool and a workflow for WDL for legacy reasons.
+
+
+Should I write a tool or a workflow?
+------------------------------------
+.. hint::
+    We generally **discourage** people from writing/registering WDL tools and may deprecate them in the future.
+
+Dockstore tools are more associated with creating/owning Docker images that are used in conjunction with a descriptor language, and Dockstore workflows are more closely tied to the descriptor files themselves. If you control the Docker image being used by your CWL program, and want versioning to be based upon that image's tags, you should probably write a tool. If you do not control the Docker image being used, and/or are not writing in CWL, you should probably write a workflow. Of course, everyone's use case is different. We encourage you to look around Dockstore for inspiration for your next bioinformatics project.
```

Comment:
> look around Dockstore

look around in Dockstore

Reply:
I think either are correct, but I'm not sure?
* "I think I left my hat in the best place to get quesadillas in Santa Cruz, I'm going to look around La Cabaña."
* "I think I left my hat in the best place to get quesadillas in Santa Cruz, I'm going to look around in La Cabaña."

Reply:
I was interpreting it like "look around the corner for my hat", which to me doesn't typically mean look in the area of the corner, but to look past the corner. But your example is a good one, so I'm good with what you have.

These are the important discussions! :)

Reply:
At our next Happy Hour we should continue this discussion, specifically as it relates to quesadillas.